docs/source/guides/staking_guide.rst

```diff
@@ -72,7 +87,8 @@ Most stakers on the Goerli testnet will complete the following steps:
 4) Stake tokens (See Below)
 5) Install another Ethereum node at the Worker instance
 6) Initialize a Worker node [:ref:`ursula-config-guide`] and bond it to your Staker (``set-worker``)
-7) Configure and run the Worker, and keep it online [:ref:`ursula-config-guide`]!
+7) Optoinally, enable restaking
```

Comment:
```suggestion
7) Optionally, enable re-staking
```

Reply:
Actually, if it's enabled by default, shouldn't it be "optionally, disable re-staking"?

Reply:
:heavy_check_mark:  At the moment, re-staking is disabled by default - see below discussion.